src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.util.DukeException;
+
+import java.util.Hashtable;
+import java.util.Scanner;
+import java.util.function.Consumer;
+
+public class Duke {
+
+    private static TaskBank taskBank = new TaskBank("data/duke.txt");
+
+    private static Hashtable<String, Consumer<String>> initCommandTable() {
+
+        Hashtable<String, Consumer<String>> ret = new Hashtable<>();
```

Comment:
what does ret mean ? perhaps the name could mean something to what the object instance is trying to perform ?